src/main.rs

```diff
@@ -121,11 +123,15 @@ async fn main() -> Result<()> {
     tokio::spawn(replicate(replication_state));
     // start unknown reaper - removes replication objects for public_keys that moved from Unkown -> Local
     tokio::spawn(reap_unknown_keys(Arc::clone(&pool), Arc::clone(&cache)));
+    // start datadog reporter
+    let (datadog_sender, datadog_receiver) = channel::<MinitraceSpans>();
```

Comment:
is there such a thing as like a /dev/null channel that just throws away anything sent to it? Trying to think of the best way to basically disable datadog properly if it isn't configured... I guess the thread will probably error when reporting to datadog and die...

Reply:
I think udp is fine with this?

Reply:
Yeah, I think at the point of sending it, it probably doesn't matter, just thinking if there was a need to even reduce the overhead of sending stuff over the channel that would just be subsequently thrown away to a non-existent udp listener, probably not a big deal though

Reply:
diving into this some more, I'm not sure that this datadog adapter from the minitrace library is actually using udp 🤔